src/app/content/components/Sidebar.tsx

```diff
@@ -4,131 +4,56 @@ import React, { Component, ComponentType } from 'react';
-import { assertString } from '../../utils';
-import * as actions from '../actions';
+import theme from '../../theme';
+import { AppState } from '../../types';
 import { isArchiveTree } from '../guards';
 import * as selectors from '../selectors';
 import { ArchiveTree, Book, Page } from '../types';
 import { scrollTocSectionIntoView, stripIdVersion } from '../utils';
 import ContentLink from './ContentLink';
 
-const sidebarOpenWidth = 300;
-const sidebarClosedWidth = 40;
-const sidebarClosedOffset = sidebarOpenWidth - sidebarClosedWidth;
+export const sidebarWidth = 33.5;
 const sidebarTransitionTime = 300;
-const sidebarControlSize = 40;
-const sidebarPadding = 10;
+const sidebarPadding = 1;
 
-const SidebarPlaceholder = styled.div<{isOpen: boolean}>`
-  overflow-x: hidden;
-  transition: all ${sidebarTransitionTime}ms;
-
-  ${(props) => !props.isOpen && css`
-    background-color: #ccc;
-    min-width: ${sidebarClosedWidth}px;
-  `}
-  ${(props) => props.isOpen && css`
-    background-color: white;
-    min-width: ${sidebarOpenWidth}px;
-  `}
-`;
-
-type StyledSidebarControlProps = React.HTMLProps<HTMLButtonElement> & {
-  isOpen: boolean,
-};
-
-const SidebarControl = styled(({isOpen, ...props}: StyledSidebarControlProps) =>
-  <FormattedMessage id={isOpen ? 'i18n:toc:toggle:opened' : 'i18n:toc:toggle:closed'}>
-    {(msg: Element | string) => {
-      const txt = assertString(msg, 'Aria label only supports strings');
-      return <button {...props}
-          aria-label={txt}
-        >
-        <span></span>
-        <span></span>
-        <span></span>
-      </button>;
-    }}
-  </FormattedMessage>
-)`
-  position: fixed;
-  height: ${sidebarControlSize}px;
-  width: ${sidebarControlSize}px;
-  margin-top: ${sidebarPadding}px;
-  background: none;
-  padding: 0;
-  border: none;
-  z-index: 1;
-  outline: none;
-  transition: all ${sidebarTransitionTime}ms;
-  transform: translateX(0px);
-
-  span {
-    width: 100%;
-    background-color: #000;
-    padding-top: 10%;
-    display: block;
-    visibility: visible;
-    opacity: 1;
-    transition: all ${sidebarTransitionTime}ms;
-  }
-
-  ${(props) => !props.isOpen && css`
-    left: ${sidebarOpenWidth - sidebarControlSize - sidebarPadding}px;
-
-    transform: translateX(-${sidebarClosedOffset}px);
-    span {
-      margin-top: 15%;
-    }
-  `}
-
-  ${(props) => props.isOpen && css`
-    // extra padding on the left for scroll bars when open
-    left: ${sidebarOpenWidth - sidebarControlSize - sidebarPadding - 10}px;
-
-    span:nth-child(1) {
-      transform: rotateZ(45deg);
-    }
-    span:nth-child(2) {
-      opacity: 0;
-    }
-    span:nth-child(3) {
-      transform: rotateZ(-45deg);
-      margin-top: -20%
-    }
-  `}
-`;
+const sidebarPositionBreak = '64em';
 
 const SidebarBody = styled.div<{isOpen: boolean}>`
-  position: fixed;
-  left: 0;
-  height: 100%;
-  padding-top: ${sidebarPadding}px;
-  width: ${sidebarOpenWidth}px;
+  top: 0;
+  height: 100vh;
```

Comment:
test with window small enough to create a horizontal scrollbar, might have to add media query on #root to set overflow:scroll on small screens, of so, make sure that doesn't break sticky positioning